test/e2e/feature/feature.go

```diff
@@ -139,6 +139,20 @@ var (
 	// TODO: document the feature (owning SIG, when to use this feature for a test)
 	HPA = framework.WithFeature(framework.ValidFeatures.Add("HPA"))
 
+	// owning-sig: sig-storage
+	// kep: https://kep.k8s.io/2680
+	// test-infra jobs:
+	// - pull-kubernetes-e2e-storage-kind-alpha-features (need manual trigger)
+	// - ci-kubernetes-e2e-storage-kind-alpha-features
+	//
```

Comment:
One question is, if no new feature is needed, then where should the relevant comments be written? Especially recording which job should run the new e2e test @pohly

Reply:
If [my plan](https://kubernetes.slack.com/archives/C09QZ4DQB/p1713379164242789) works out, you don't need a new job. Your new test should start to run in `ci-kubernetes-e2e-kind-alpha-features` automatically and `*-e2e-storage-kind-alpha-features` should not be necessary for it.

If we don't have a `pull-kubernetes-e2e-kind-[alpha|beta]-features` yet, then we should add them to allow triggering them manually when adding new alpha or beta tests or when modifying features that they cover.

/cc @BenTheElder @aojea

Reply:
/hold

Reply:
/hold cancel

To quickly promote this feature to Beta, we decide to merge it. I will raise another PR to remove unnecessary code once https://github.com/kubernetes/kubernetes/pull/124350 is merged.

/cc @pohly @xing-yang 

rebased. please review it again. thank you.

Reply:
but the k8s robot doesn't remove the `needs-rebase` label.

Reply:
@pohly I approved.  Thanks.

Reply:
/approve